data/benchmarking.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @title Benchmarking
+ * @difficulty beginner
+ * @tags cli
+ * @run deno bench --allow-net <url>
```

Comment:
Is `--allow-net` unnecessary?

Reply:
The async method that you also commented on used to be fetch before I decided not to do that, so this was necessary then. Nice catch!